docs/api/middleware/cors.md

```diff
@@ -35,52 +35,72 @@ app.Use(cors.New(cors.Config{
 }))
```

Comment:
btw: docs are auto-generated and shouldn't be modified by hand :)

Reply:
will not be auto-generated, have already cleared it with him

Reply:
Sorry for the confusing, was under the impression we auto-gen the docs nowadays.

I've opened https://github.com/gofiber/fiber/issues/2401 as a feature request for this.